main.go

```diff
@@ -674,14 +674,14 @@ func syncIfOSHasEnterpriseRepo(conf *config, gpr *github.PullRequestEvent) error
 	// don't include pull request events with the "synchronize" action.
 	if gpr.GetAction() == "closed" && pr.GetMerged() {
 
-		// Only sync on Merges to master or release branches and
-		// verify release branches.
+		// Only sync on Merges to master, release and feature branches.
+		// Verify release branches.
 		branch := pr.GetBase()
 		if branch == nil {
 			return fmt.Errorf("syncIfOSHasEnterpriseRepo: Failed to get the base-branch of the PR: %v", branch)
 		}
 
-		syncBranches := regexp.MustCompile(`(master|[0-9]+\.[0-9]+\.x)`)
+		syncBranches := regexp.MustCompile(`(master|[0-9]+\.[0-9]+\.x|feature-.+)`)
```

Comment:
Question:
Just to double check here:
Is this the right syntax for `feature` branches?

Reply:
@oleorhagen Yes, you can see it for example in [the pipeline](https://github.com/mendersoftware/mendertesting/blob/master/.gitlab-ci-check-docker-build.yml#L87).

(I'll slightly rework this PR to re-use a recently introduced constant for this)